template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/Microsoft.DotNet.Common.ProjectTemplates.7.0.csproj

```diff
@@ -10,6 +10,7 @@
         <!-- .NET 6 template package is shipped with the preview / RC since .NET 6 is not released yet -->
         <!-- if the package should be shipped for servicing, set <IsPackable> to true for explicit version here -->
         <IsPackable>false</IsPackable>
+        <IsPackable Condition="'$(PreReleaseVersionLabel)' == 'alpha'">true</IsPackable>
```

Comment:
This is no longer needed. I'll remove this block. This was previously needed for older templates version not to release them with preview as it yielded too many releases without changes. Now since we kept only current version this is no longer needed.